server/src/main/java/org/opensearch/action/admin/cluster/node/info/NodeAnalyzers.java

```diff
@@ -0,0 +1,267 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.node.info;
+
+import org.opensearch.common.Nullable;
+import org.opensearch.common.collect.Tuple;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.service.ReportingService;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.index.analysis.AnalysisRegistry;
+import org.opensearch.plugins.AnalysisPlugin;
+import org.opensearch.plugins.PluginInfo;
+import org.opensearch.plugins.PluginsService;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import static java.util.Collections.unmodifiableMap;
+import static java.util.Collections.unmodifiableSortedSet;
+
+/**
+ * Information about node analyzers and other text analysis components.
+ *
+ * @opensearch.internal
+ */
+public class NodeAnalyzers implements ReportingService.Info {
+
+    @Nullable
```

Comment:
Please remove `@Nullable`, those fields cannot be null in current implementation